### PR TITLE
Hw5 photoshop

### PR DESCRIPTION
**Pull request by Lina-Karyzhska**

### CODE REVIEW

File: hw5/style.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+body {
+    font-family: 'Open Sans', sans-serif;
+    background-color: #121116;
+    color: white;
+}
+
+.wrapper {
+    margin: auto;
+    background-color: #121116;
```
Comment (idovbush) [inline hw5/style.css:14]: you have already made that changes to body, here it's unnecessary because the color is equal

Comment (Lina-Karyzhska) [reply hw5/style.css:14]: forgot to remove the color in body

**Review by idovbush — APPROVED** [review]

File: hw5/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <ul>
+                <li>
+                    <a href="#">
+                        <img src="Facebook.png" alt="Facebook icon">
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <img src="Tweter.png" alt="Tweter icon">
```
[the remark this answers is not in the document]
Comment (Lina-Karyzhska) [reply hw5/index.html:34]: it is the name of an icon, so i used it

File: hw5/style.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    letter-spacing: 3px;
+    font-weight: 300;
+}
+
+nav {
+    margin-top: 20px;
+}
+
+li::before {
+    content: none;
```
[the remark this answers is not in the document]
Comment (Lina-Karyzhska) [reply hw5/style.css:40]: i used li for icons, and i didn't want them to have a marker

**Pull request merged**